### PR TITLE
fixed some android snippets and bumped sentry-java

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/_data/platforms.yml [hunk 1]
```diff
@@ -93,10 +93,10 @@
       support_level: production
       type: framework
       name: Android
-      doc_link: /clients/java/modules/android/
-      wizard_parent: java
+      doc_link: /platforms/android/
+      wizard_parent: android
       wizard:
-        - _documentation/clients/java/integrations.md#android
+        - _documentation/platforms/android/#integrating-the-sdk
```
Comment (HazAT) [inline src/_data/platforms.yml]: Look for places where wizard: true
I think this is how it works now.
Compare to php or something else.

Also there is <!— Wizard android —> double check so we are not missing something

Comment (marandaneto) [reply src/_data/platforms.yml]: I am a bit lost here, not sure how to test it either, all versions (other platforms) are a bit different and this one was a client/integration and now its own platform.
how do I test it?

Comment (cameronmcefee) [reply src/_data/platforms.yml]: My environment is in an inoperable state currently so I can't verify this for you, but to compile this, run:

```
JEKYLL_ENABLE_PLATFORM_API=true bin/server
```

Check the output of _site for a folder called _platforms and then look to see if your new stuff exists in there. This is one of the most opaque parts of the docs, so it's difficult to deal with. Code is here: https://github.com/getsentry/sentry-docs/blob/master/src/_plugins/platform_api.rb

**Comment by MimiDumpling** [comment]
@marandaneto Let me know if Cameron's suggestion works. If not, we can still deploy the docs without the wizard and add the wizard later.

File: src/_data/platforms.yml (hunk 1 above)
Comment (marandaneto) [reply src/_data/platforms.yml]: thanks for the tips, the command didn't work though :(
`NoMethodError: undefined method `content' for nil:NilClass`
I'll merge without the wizard and figure it out later on, thanks

**Pull request merged**